src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -587,6 +577,7 @@ def run(self) -> None:
                 self._mca.significant_coordinate_clusters
             )
             count = 0
+            subclusters = []
```

Comment:
It looks like we're initializing the subclusters list twice, do we need to define it once before all the different cluster code branching?